### ISSUE TITLE
oplik0 does not have my credit card

### ISSUE THREAD

**Issue by julianlam** [oplik0/github-templates-experiments#5]
### NodeBB version

v2.2.0

### NodeBB git hash

abcduni238dhiunwed

### NodeJS version

0.9

### Installed NodeBB plugins

no plugins installed

### Database type

Redis

### Database version

1

### Exact steps to cause this issue

1. I asked oplik0 to fetch my lunch #4 
2. ... but he does not have my credit card

### What you expected

A hot lunch ready at my table

### What happened instead

Nothing, he couldn't pay for it.

### Anything else?

Maybe we can send money his way in the form of a balance so he does not need my credit card.

**Comment by oplik0** [comment]
Ah, sorry for labels here - forgot to change that

**Comment by julianlam** [comment]
No worries, I presume it was just an outdated config :smile:

**Comment by oplik0** [comment]
Yeah, the config only works on main branch so I wasn't using the NodeBB repo but this quickly set up one and as such all changes had to be copied over between the fork and here.
The labels on the PR are correct, just didn't move them here.
